src/main/java/com/amihaiemil/docker/RtContainer.java

```diff
@@ -71,8 +71,8 @@ public void start() throws IOException {
         final HttpResponse response = this.client.execute(start);
         final int status = response.getStatusLine().getStatusCode();
         if(status != HttpStatus.SC_NO_CONTENT) {
-            throw new IllegalStateException(
-                "Container#start() expected status 204, but got " + status
+            throw new UnexpectedResponseException(
+                "Containers#start()", status, HttpStatus.SC_NO_CONTENT
```

Comment:
@amihaiemil the first argument to ctor of `UnexpectedResponseException` is supposed to be the endpoint